package.json

```diff
@@ -14,6 +14,9 @@
     "type": "git",
     "url": "git+https://github.com/blackbaud/sky-pages-out-skyux2.git"
   },
+  "engines": {
+    "node": ">= 6.0.0"
```

Comment:
We might want to do >= 6.3.0 since I think there were some changes related to debugging in that version we may be relying on.